### ISSUE TITLE
Incorrect type names given with type.FullName.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1293]
[Dot Net Fiddle](https://dotnetfiddle.net/FRJ6c0)
[Bridge Fiddle](http://live.bridge.net/#39d26b19bcaae263caf3d65c0e676fa2)
### Expected

System.Int32
System.Single
System.Double
System.Decimal

### Actual

Bridge.Int32
Bridge.Single
Bridge.Double
Bridge.Decimal

### Steps To Reproduce

```csharp
public class App
{
    [Ready]
    public static void Main()
    {
        int v = 0;
        float v2 = 1;
        double v3 = 2;
        decimal v4 = 3;
        Console.WriteLine(v.GetType().FullName);
        Console.WriteLine(v2.GetType().FullName);
        Console.WriteLine(v3.GetType().FullName);
        Console.WriteLine(v4.GetType().FullName);
    }
}
```

**Comment by geoffreymcgill** [comment]
Related to #1248, #1292 and #1184.

**Comment by LeonidVeriga** [comment]
This particular case has been fixed by #1337
